lib/merchant_repository.rb

```diff
@@ -47,7 +44,7 @@ def create(attributes)
   end
 
   def update(id, attributes)
-    find_by_id(id).update(attributes)
+    find_by_id(id)&.update(attributes)
```

Comment:
this is what hound told me to do instead of doing `unless find_by_id(id).nil?` or `if  !find_by_id(id).nil?`
Style/SafeNavigation: Use safe navigation (&.) instead of checking if an object exists before calling the method.